ambry-tools/src/main/java/com.github.ambry/tools/perf/rest/NettyPerfClient.java

```diff
@@ -446,8 +462,15 @@ public void channelRead0(ChannelHandlerContext ctx, HttpObject in) {
         perfClientMetrics.timeToFirstResponseChunkInMs.update(responseReceiveStart);
         logger.trace("Response receive has started on channel {}. Took {} ms", ctx.channel(), responseReceiveStart);
         response = (HttpResponse) in;
-        if (response.status() != HttpResponseStatus.OK) {
-          logger.error("Got Response code {} and headers were {}", response.status().code(), response.headers());
+        if (response.status().code() >= 200 && response.status().code() < 300) {
+          logger.trace("Request succeeded");
+          if (response.headers().contains("Location")) {
+            logger.info(response.headers().get("Location"));
+          }
+        } else if (response.status().code() >= 300 && response.status().code() < 400) {
+          logger.warn("Redirection code {} and headers were {}", response.status().code(), response.headers());
+        } else {
+          logger.error("Response error code {} and headers were {}", response.status().code(), response.headers());
```

Comment:
any reason to log these messages?

Reply:
Recording 50x error code might be useful to remind us there is something wrong on server side. 40x can drive us to double check if server behaves as expected.